dispatcher/frontend-ui/src/constants.js

```diff
@@ -295,6 +295,10 @@ function get_timezone_details() {
   return {tz: dt.zoneName, offset: dt.o, offsetstr: offsetstr}
 }
 
+function delay(time) {
```

Comment:
This is unclear. `time` means nothing. Is this milliseconds?

Reply:
Fixed